src/main/java/org/minions/devfund/katerinaanzoleaga/CountCharacter.java

```diff
@@ -0,0 +1,42 @@
+package org.minions.devfund.katerinaanzoleaga;
+
+/**
+ *
+ * Class to count the number of occurrences of a char in a string.
+ */
+public final class CountCharacter {
+
+    /**
+     * Private constructor.
+     */
+    private CountCharacter() {
+
+    }
+
+    /**
+     * This is the metod to get charCount that will do the counting of the occurrences of c ub str.
+     * @param str The string
+     * @param c The character to count
+     * @return int how many times c appears on str
+     */
+    public static int charCount(final String str, char c) {
```

Comment:
This may be refactored like this.
```java
    public static int charCount(final String str, char c) {
        // your code here
        int count = 0;
        int i = 0;
        int j = str.length() - 1;
        char upperCase = Character.toUpperCase(c);
        while (j >= i) {
            if (Character.toUpperCase(str.charAt(i)) == upperCase
                    || Character.toUpperCase(str.charAt(j)) == upperCase) {
                count++;
            }
            i++;
            j--;
        }
        return count;
    }
```